docs/troubleshooting.md

```diff
@@ -8,4 +8,9 @@ Infrastructure logs are visible from the `Global` tenant and require `administra
 `kube:admin` by default does not have the correct permissions to be given the admin role.   See the [access control](access-control.md) documentation for additional informations.  You may grant the permissions by:
 ```
 oc adm policy add-cluster-role-to-user cluster-admin kube:admin
-```
\ No newline at end of file
+```
+
+## Elasticsearch
+
+### Why do I receive the alert `ElasticsearchHighFileDescriptorUsage`
+The cause of this alert is due to [insufficient file descriptors on the node] (https://www.elastic.co/guide/en/elasticsearch/reference/current/file-descriptors.html).
```

Comment:
I suggest to add a new markdown file named `alerts.md` and structure it like this:
https://gitlab.cee.redhat.com/observatorium/configuration/-/blob/master/docs/sop/observatorium.md

This way you can link each alert to its subsection in `alerts.md`. WDYT?

Reply:
Agreed. Thanks.